config/config.go

```diff
@@ -57,6 +64,33 @@ func NewLocal(cfgPath, basepath string) (*Local, error) {
 		return nil, fmt.Errorf("failed to unmarshal local configuration (%s): %w", cfgPath, err)
 	}
 
+	// Check that origin matches issue tracker's format
```

Comment:
Could you pull latest master & move this chunk into the `Validate` function for consistency? 

See #46

Reply:
resolved in 78a7217